paddle/majel/test/place_test.cpp

```diff
@@ -1,6 +1,6 @@
-#include "gtest/gtest.h"
 #include "majel/place.h"
 #include <sstream>
+#include "gtest/gtest.h"
```

Comment:
Sorry that I forgot to reorder the include files following https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes. You can do that in this PR.

I think the right order is:

```cpp
#include "majel/place.h"

#include <sstream>

#include "gtest/gtest.h"
```